WidgetExtension/Variants/LatestFollowers/LatestFollowersWidgetView.swift

```diff
@@ -0,0 +1,140 @@
+                        .truncationMode(.tail)
+     
+                    }
+                    Spacer()
+                }
+            }
+            Spacer()
+            Text(L10n.Widget.LatestFollowers.lastUpdate(dateFormatter.string(from: lastUpdate)))
+                .font(.caption2)
+                .foregroundColor(.secondary)
+        }
+        .padding(.horizontal, 20)
+        .padding(.vertical, 16)
+    }
+    
+    private func viewForMediumWidget(_ accounts: [LatestFollowersEntryAccountable], lastUpdate: Date) -> some View {
+        VStack(alignment: .leading) {
+            HStack {
+                Text(L10n.Widget.LatestFollowers.title)
+                    .font(.system(size: UIFontMetrics.default.scaledValue(for: 16)))
+                Spacer()
+                Image("BrandIconColored")
+            }
+            
+            ForEach(accounts, id: \.acct) { account in
+                HStack {
+                    if let avatarImage = account.avatarImage {
+                        Image(uiImage: avatarImage)
+                            .resizable()
+                            .frame(width: 32, height: 32)
+                            .cornerRadius(5)
+                    }
+                    VStack(alignment: .leading) {
+                        
+                        HStack {
+                            Text(account.displayNameWithFallback)
+                                .font(.footnote.bold())
+                                .lineLimit(1)
+                                .truncationMode(.tail)
+                            
+                            Text("@\(account.acct)")
+                                .font(.caption2)
+                                .foregroundColor(.secondary)
+                                .lineLimit(1)
+                                .truncationMode(.tail)
+                        }
+                        
+                        Text(account.noteWithoutHtmlTags ?? "")
+                            .font(.caption)
+                            .lineLimit(1)
+                            .truncationMode(.tail)
+                    }
+                    Spacer()
+                }
+            }
+            Spacer()
+            Text(L10n.Widget.LatestFollowers.lastUpdate(dateFormatter.string(from: lastUpdate)))
+                .font(.caption2)
+                .foregroundColor(.secondary)
+        }
+        .padding(.horizontal, 20)
+        .padding(.vertical, 16)
+    }
+}
+
+private extension LatestFollowersEntryAccountable {
+    var noteWithoutHtmlTags: String? {
+        do {
+            let regex =  "<[^>]+>"
+            let expr = try NSRegularExpression(pattern: regex, options: NSRegularExpression.Options.caseInsensitive)
+            let result = expr.stringByReplacingMatches(in: note, options: [], range: NSMakeRange(0, note.count), withTemplate: "")
+            return result
+        } catch {
+            return nil
+        }
+    }
```

Comment:
Can you please say a few words about this, i.e. why it's necessary?

Reply:
It's a regex to remove all HTML tags, unfortunately we can use `MetaLabelRepresentable` / `MetaLabel` as it's backed by UIKit (which is not available in Widgets). I'm removing all HTML tags from the bio description. Unfortunately this is not perfect, e.g. Emojis are not being resolved. I'll create an issue for this as well, if you think it's fine that we can go with this for the first iteration.

Reply:
I'll add a code-comment regarding this as well.  Files https://github.com/mastodon/mastodon-ios/issues/921 for this.